[2025-09-15T23:45:01.432+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-15T23:45:01.516+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T21:30:00+00:00 [queued]>
[2025-09-15T23:45:01.543+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T21:30:00+00:00 [queued]>
[2025-09-15T23:45:01.548+0200] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-15T23:45:01.581+0200] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_customers> on 2025-09-15 21:30:00+00:00
[2025-09-15T23:45:01.599+0200] {standard_task_runner.py:64} INFO - Started process 307 to run task
[2025-09-15T23:45:01.610+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_api_to_bi', 'extract_customers', 'scheduled__2025-09-15T21:30:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_bi.py', '--cfg-path', '/tmp/tmpptiy3ilf']
[2025-09-15T23:45:01.617+0200] {standard_task_runner.py:91} INFO - Job 9: Subtask extract_customers
[2025-09-15T23:45:01.776+0200] {task_command.py:426} INFO - Running <TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T21:30:00+00:00 [running]> on host b5272c808890
[2025-09-15T23:45:02.465+0200] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data' AIRFLOW_CTX_DAG_ID='etl_api_to_bi' AIRFLOW_CTX_TASK_ID='extract_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-15T21:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-15T21:30:00+00:00'
[2025-09-15T23:45:02.466+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-15T23:45:02.600+0200] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-09-15T23:45:02.603+0200] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-09-15T23:45:02.657+0200] {python.py:237} INFO - Done. Returned value was: 1
[2025-09-15T23:45:02.661+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-15T23:45:02.723+0200] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_api_to_bi, task_id=extract_customers, run_id=scheduled__2025-09-15T21:30:00+00:00, execution_date=20250915T213000, start_date=20250915T214501, end_date=20250915T214502
[2025-09-15T23:45:02.776+0200] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-15T23:45:02.824+0200] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-15T23:45:02.827+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-09-15T23:57:44.907+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-15T23:57:45.049+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T21:30:00+00:00 [queued]>
[2025-09-15T23:57:45.088+0200] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T21:30:00+00:00 [queued]>
[2025-09-15T23:57:45.094+0200] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-15T23:57:45.148+0200] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_customers> on 2025-09-15 21:30:00+00:00
[2025-09-15T23:57:45.173+0200] {standard_task_runner.py:64} INFO - Started process 211 to run task
[2025-09-15T23:57:45.198+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_api_to_bi', 'extract_customers', 'scheduled__2025-09-15T21:30:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/etl_api_to_bi.py', '--cfg-path', '/tmp/tmp24uux4lk']
[2025-09-15T23:57:45.207+0200] {standard_task_runner.py:91} INFO - Job 2: Subtask extract_customers
[2025-09-15T23:57:45.420+0200] {task_command.py:426} INFO - Running <TaskInstance: etl_api_to_bi.extract_customers scheduled__2025-09-15T21:30:00+00:00 [running]> on host b91cbf0a6013
[2025-09-15T23:57:46.304+0200] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data' AIRFLOW_CTX_DAG_ID='etl_api_to_bi' AIRFLOW_CTX_TASK_ID='extract_customers' AIRFLOW_CTX_EXECUTION_DATE='2025-09-15T21:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-15T21:30:00+00:00'
[2025-09-15T23:57:46.305+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-15T23:57:46.505+0200] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-09-15T23:57:46.508+0200] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-09-15T23:57:46.791+0200] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2025-09-15T23:57:47.236+0200] {python.py:237} INFO - Done. Returned value was: 1000
[2025-09-15T23:57:47.242+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-15T23:57:47.313+0200] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_api_to_bi, task_id=extract_customers, run_id=scheduled__2025-09-15T21:30:00+00:00, execution_date=20250915T213000, start_date=20250915T215745, end_date=20250915T215747
[2025-09-15T23:57:47.382+0200] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-15T23:57:47.449+0200] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-15T23:57:47.454+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
